Apps/phsentinelone/sentinelone.json

```diff
@@ -446,6 +448,480 @@
             },
             "versions": "EQ(*)"
         },
+        {
+            "action": "abort scan",
+            "description": "Abort scan for threats",
+            "type": "generic",
+            "identifier": "abort_scan",
+            "read_only": false,
+            "parameters": {
+                "ip_hostname": {
+                    "description": "The hostname of an endpoint to abort scan",
```

Comment:
Correct the description. It should be like this (you missed to put IP)
"The IP or hostname of an endpoint to abort scan" 
Do this for all the applicable actions.